javascript/builtins/BigInt.json

```diff
@@ -0,0 +1,67 @@
+{
+  "javascript": {
+    "builtins": {
+      "BigInt": {
+        "__compat": {
+          "mdn_url": " https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/BigInt",
+          "support": {
+            "chrome": {
+              "version_added": "67"
+            },
+            "chrome_android": {
+              "version_added": "67"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "65",
+			        "flags": [
+			        {
+					      "type": "preference",
+					      "name": "javascript.options.bigint",
+					      "value_to_set": "true"
+				      }
+			        ]
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "nodejs": {
+              "version_added": "10.4.0"
+            },
+            "opera": {
+              "version_added": "54"
+            },
+            "opera_android": {
+              "version_added": "54"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "67"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+    }

```

Comment:
The indentation of these last few braces got a bit jumbled. Could you tidy this up?